### PR TITLE
Added MacOs Terminal application support

### PR DESCRIPTION
**Pull request by vasste**

### CODE REVIEW

File: src/org/turbanov/execution/cmd/InCmdRunner.java [hunk 1]
```diff
@@ -1,17 +1,9 @@
...
 import com.intellij.execution.JavaTestConfigurationBase;
 import com.intellij.execution.application.ApplicationConfiguration;
-import com.intellij.execution.configurations.GeneralCommandLine;
-import com.intellij.execution.configurations.JavaCommandLineState;
-import com.intellij.execution.configurations.JavaParameters;
-import com.intellij.execution.configurations.ParametersList;
-import com.intellij.execution.configurations.RunProfile;
-import com.intellij.execution.configurations.RunProfileState;
-import com.intellij.execution.configurations.RunnerSettings;
+import com.intellij.execution.configurations.*;
```
Comment (turbanoff) [inline src/org/turbanov/execution/cmd/InCmdRunner.java]: Let's not use start import

File: src/org/turbanov/execution/cmd/InCmdRunner.java [hunk 2]
```diff
@@ -111,39 +108,28 @@ protected RunContentDescriptor doExecute(@NotNull RunProfileState runProfileStat
...
     }
 
-    private static void runInExternalCmd(String classPathPathsString, GeneralCommandLine generalCommandLine, String workingDirectory, String commandLine) throws ProcessNotCreatedException {
-        String[] command;
-        if (SystemInfo.isWindows) {
-            command = new String[]{"cmd.exe", "/C", "\"start cmd.exe /K \"" + commandLine + "\"\""};
-        } else {
-            List<String> parsedLine = ParametersListUtil.parse(commandLine, false, true);
-            ArrayList<String> result = new ArrayList<>(parsedLine);
-            result.add(0, "gnome-terminal");
```
Comment (turbanoff) [inline src/org/turbanov/execution/cmd/InCmdRunner.java:141]: can you please bring back `gnome-terminal` support. Use it for in case of Linux

**Review by turbanoff — CHANGES_REQUESTED** [review]

File: src/org/turbanov/execution/cmd/InCmdRunner.java (hunk 1 above)
Comment (vasste) [reply src/org/turbanov/execution/cmd/InCmdRunner.java]: Removed

File: src/org/turbanov/execution/cmd/InCmdRunner.java (hunk 2 above)
Comment (vasste) [reply src/org/turbanov/execution/cmd/InCmdRunner.java:141]: Returned.

**Pull request merged**